components/landing/MobileLanding.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <section class="is-relative landing-search is-flex is-align-items-center">
+    <img
+      class="landing-image"
+      src="/landing-mobile-shape.svg"
+      alt="landing page background" />
+    <div
+      class="is-flex is-flex-direction-column is-align-items-center search-info">
+      <h1
+        class="title is-size-3 has-text-weight-bold has-text-centered is-flex is-flex-wrap-wrap is-justify-content-center is-align-items-center">
+        {{ $t('search.landingTitle1') }}
+      </h1>
+      <h1
+        class="subtitle is-size-3 has-text-weight-bold has-text-centered is-capitalized">
+        {{ $t('search.landingTitle2') }}
+      </h1>
+      <h1 class="title is-size-3 has-text-weight-bold has-text-centered">
+        {{ $t('search.landingTitle3') }}
+      </h1>
```

Comment:
multiple `h1` you could try to wrap it in a `h1` and use `span` for text class

Reply:
done